### DISCUSSION TITLE
Cypress.on uncaught:exception send logs to console.log only and NOT command log

### DISCUSSION THREAD

**Discussion by dlavigneNBguy** [cypress-io/cypress#23463] [Questions and Help]
Hello,

We have this in our support/index.js as per many other recommendations:
`const resizeObserverLoopErrRe = /^[^(ResizeObserver loop limit exceeded)]/
Cypress.on('uncaught:exception', (err) => {
  /* returning false here prevents Cypress from failing the test */
  if (resizeObserverLoopErrRe.test(err.message)) {
    return false
  }
  return true
})`
But on runs, it still will output "**(uncaught exception)Error: ResizeObserver loop limit exceeded**" so the reports and command logs every time it is hit.  We know about it and just don't want to see it in the command log or reports.  I've tried many different ways to stop it from going to command log but nothing worked.  We just want it to go to console log only (for this particular message)

ps, it does retry twice, as a bonus, how can we disconnect the retries on that particular Cypress.on being used there.

Thanks

DL

**Comment by jjhelguero** [comment]
It looks like you have a `return true` outside of your if statement. Have you removed that and tried again?

**Reply by dlavigneNBguy** [reply]
well everything works fine, it just writes to our command log, which is what I don't want.  We have the return true to satisfy our eslint rules really.  It required a return from a => promise.
We just want a way that is does not come out in our command log or reports.  Just to console.log
